css/comp.css

```diff
@@ -41,3 +42,253 @@
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 2rem;
+}
+
+.badge-icon {
+  color: var(--darker);
+  font-size: 10rem !important;
+}
+
+.badge-st {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  border-radius: 50%;
+  border: none;
+  font-size: 1.2em;
+  right: 1em;
+  bottom: 0;
+  width: 1.8em;
+  height: 1.8em;
+}
+
+.badge-primary {
+  background-color: var(--dark-primary);
+  color: white;
+}
+.badge-danger {
+  background-color: var(--danger);
+  color: white;
+}
+
+.badge-warning {
+  background-color: var(--warning);
+  color: var(--darker);
+}
+
+.badge-success {
+  background-color: var(--success);
+  color: var(--darker);
+}
+
+.badge-dark {
+  background-color: var(--darker);
+  color: var(--light);
+}
+
+.badge-default {
+  background-color: var(--light);
+  color: black;
+}
+
+/* ---------------------------------------------- Buttons */
+
+.btn {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 0.5rem;
+  font-size: 2rem;
+  padding: 1rem 1.5rem;
+  justify-content: center;
+  border: none;
+  cursor: pointer;
+  border-radius: 0.5rem;
+  transition: 350ms;
+}
+
+.btn:hover {
+  background-color: var(--light-primary);
+}
+
+.btn-primary {
+  display: inline-flex;
+  justify-content: center;
+  margin: 0 0.5rem;
+  font-size: 2rem;
+  background-color: white;
+  color: var(--dark-primary);
+  border: 0.1rem solid var(--dark-primary);
+  padding: 1rem 1.5rem;
+  justify-content: center;
+  cursor: pointer;
+  border-radius: 0.5rem;
+  transition: 350ms;
+}
+
+.btn-primary:hover {
+  background-color: var(--dark-primary);
+  color: white;
+}
+
+.btn-warning {
+  display: inline-flex;
+  justify-content: center;
+  margin: 0 0.5rem;
+  font-size: 2rem;
+  background-color: white;
+  color: var(--warning);
+  border: 0.1rem solid var(--warning);
+  padding: 1rem 1.5rem;
+  justify-content: center;
+  cursor: pointer;
+  border-radius: 0.5rem;
+  transition: 350ms;
+}
+
+.btn-warning:hover {
+  background-color: var(--warning);
+  color: white;
+}
+
+.btn-danger {
+  display: inline-flex;
+  justify-content: center;
+  align-content: center;
+  margin: 1rem;
+  font-size: 2rem;
+  background-color: white;
+  color: var(--danger);
+  border: 0.1rem solid var(--danger);
+  padding: 1rem 1.5rem;
+  justify-content: center;
+  cursor: pointer;
+  border-radius: 0.5rem;
+  transition: 350ms;
+}
+
+.btn-danger:hover {
+  background-color: var(--danger);
+  color: white;
+}
+
+/* -------------Button link */
+
+.btn-link {
+  font-size: 2rem;
+  margin: 0 1rem;
+  padding-bottom: 0.2rem;
+  border-bottom: 0.05rem solid white;
+  transition: 360ms;
+}
+
+.btn-link:hover {
+  border-bottom: 0.05rem solid var(--darker);
+}
+
+.btn-link-primary {
+  color: var(--dark-primary);
+  font-size: 2rem;
+  margin: 0 1rem;
+  padding-bottom: 0.2rem;
+  border-bottom: 0.05rem solid white;
+  transition: 360ms;
```

Comment:
Same here you can directly add `btn-link` in HTML and remove these properties from this class.